sdk/tables/azure-data-tables/azure/data/tables/_table_service_client.py

```diff
@@ -273,7 +273,7 @@ def query_tables(self, query_filter: str, **kwargs) -> ItemPaged[TableItem]:
         )
 
     @distributed_trace
-    def list_tables(self, **kwargs) -> ItemPaged[TableItem]:
+    def list_tables(self, *, results_per_page: Optional[int] = None, **kwargs) -> ItemPaged[TableItem]:
```

Comment:
It might be worth it to check the other methods on the client to see if there are other instances of keywords in docstrings, but missing in signatures. The pylint custom rule has a bug where it is only reporting one error per client.

Reply:
Checked all methods in async and sync base_client.py, table_service_client.py and table_client.py.